### ISSUE TITLE
Blue / Black contrast on the address bar autocomplete highlight is poor

### ISSUE THREAD

**Issue by kbrosnan** [mozilla-mobile/fenix#860]
## Steps to reproduce
* in normal browsing mode
* tap on the address bar
* type something that matches your autocomplete (ex. wiki)

### Expected behavior
* readable autocomplete highlight

### Actual behavior
* difficult to read the text in the autocomplete highlight

### Device information

* Android device: Any
* Fenix version: March 5 build

**Comment by ekager** [comment]
In #581 UX gave us this color, maybe this color needs to be different for more contrast for accessibility? cc @AmyYLee

**Comment by Cheap-Skate** [comment]
I second this issue, my vision is not great and I find the text hard to read.

**Comment by kbrosnan** [comment]
Looking at desktop and the Photon Design docs suggest pairing [Photon Blue-50 with white text](https://firefoxux.github.io/photon/visuals/color.html#blue).

**Comment by mheubusch** [comment]
@AmyYLee can you look at this contrast issue?

**Comment by AmyYLee** [comment]
Can you please use Violet 70 (#592ACB) with white text on top. This is our accent colour on light theme. Thanks

**Comment by ekager** [comment]
Sorry @AmyYLee to clarify - the suggestion text being highlighted for autocomplete should be white text with the violet highlight, and the text not being highlighted should be black? 
![Screenshot_20190407-200712](https://user-images.githubusercontent.com/25442310/55696259-ff8ab380-5970-11e9-9e42-65fe990f4ab7.png)

**Comment by Verdi** [comment]
Sorry about changes...
@shorlander and I talked about this. We should use accent @ 36% and don't invert the text.
![highlight](https://user-images.githubusercontent.com/107992/55763735-d7ad5580-5a2d-11e9-9520-da366cdfef58.png)

**Comment by ekager** [comment]
Got it, thanks!
